config/locales/doorkeeper.en.yml

```diff
@@ -89,6 +89,7 @@ en:
         invalid_request:
           missing_param: 'Missing required parameter: %{value}.'
           request_not_authorized: Request need to be authorized. Required parameter for authorizing request is missing or invalid.
+          offline_access_only: The offline_access scope can only be used with other scopes.
```

Comment:
This is meant to be the error shown from `pre_auth` check in `app/controllers/oauth/authorizations_controller.rb` but it's not working right now, I'm not 100% sure why.